packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -275,7 +268,7 @@ export default class TinyMCE extends Component {
 			this.removeInternetExplorerInputFix = null;
 		}
 
-		if ( editorNode && needsInternetExplorerInputFix( editorNode ) ) {
```

Comment:
A `ref` callback function is called with `null` as the argument when the component unmounts. I suspect this is why that part of the condition had been there. Since `applyInternetExplorerInputFix` immediately accesses properties on `editorNode`, I assume it's now throwing errors when passed the `null` value ?

Reply:
Pressing Backspace in a new paragraph:

![image](https://user-images.githubusercontent.com/1779930/49044347-792cbc80-f19b-11e8-8f78-8d5a9810dcef.png)